### PR TITLE
stub + nf-test

### PR DESCRIPTION
**Pull request by sateeshperi**

### CODE REVIEW

**Comment by sateeshperi** [comment]
@nf-core-bot fix linting

File: modules/nf-core/nucmer/tests/main.nf.test [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                    file(process.out.delta[0][1]).readLines()[2..-1],
+                    file(process.out.coords[0][1]).readLines()[2..-1],
```
Comment (maxulysse) [inline modules/nf-core/nucmer/tests/main.nf.test:29]: Can we get the md5 of these lines instead of the lines themselves?

**Review by maxulysse — APPROVED** [review]
`file(process.out.delta[0][1]).readLines()[2..-1]`
Just wondering here if we can md5 this instead of having this content in the snapshot, but that's ok for me anyway.
:rocket:

**Comment by sateeshperi** [comment]
not able to get md5 easily...lets go with this for now as it is still deterministic

**Comment by maxulysse** [comment]
> not able to get md5 easily...lets go with this for now as it is still deterministic

I meant something like:
```
file(process.out.delta[0][1]).readLines()[2..-1].md5
```

**Comment by sateeshperi** [comment]
ya i tried `file(process.out.delta[0][1]).readLines()[2..-1].md5()` and it didn't work

**Pull request merged**